index.js

```diff
@@ -1,5 +1,74 @@
 'use strict';
 
+var colorConvert = require('color-convert');
+var supportsColor = require('supports-color');
+
+supportsColor = supportsColor || {};
+supportsColor.level = supportsColor.level || 0;
+
+colorConvert.rgb.rgb = function rgb (rgb) {
+	return rgb;
+};
+
+function getConverter (color) {
```

Comment:
[I'm still confused](https://github.com/chalk/ansi-styles/pull/13#discussion_r35481365). The [line you're referring to](https://github.com/chalk/ansi-styles/pull/13#discussion-diff-35481365R13) has a space between `getConverter` and `()`. It's not a statement (i.e. not being assigned to something), so it's inarguably necessary to have a function name. What's the problem?